AnnoDesigner/AnnoCanvas.xaml.cs

```diff
@@ -1522,21 +1576,30 @@ protected override void OnKeyDown(KeyEventArgs e)
         /// <summary>
         /// Checks whether the user is pressing the control key.
         /// </summary>
-        /// <returns></returns>
+        /// <returns><c>true</c> if the control key is pressed, otherwise <c>false</c>.</returns>
         private static bool IsControlPressed()
         {
-            return Keyboard.Modifiers.HasFlag(ModifierKeys.Control) || Keyboard.Modifiers.HasFlag(ModifierKeys.Shift);
+            return Keyboard.Modifiers.HasFlag(ModifierKeys.Control);
         }
 
         /// <summary>
         /// Checks whether the user is pressing the shift key.
         /// </summary>
-        /// <returns></returns>
+        /// <returns><c>true</c> if the shift key is pressed, otherwise <c>false</c>.</returns>
         private static bool IsShiftPressed()
         {
             return Keyboard.Modifiers.HasFlag(ModifierKeys.Shift);
```

Comment:
Pretty much the same as `IsControlPressed()`, but using `Keyboard.Modifiers & ModifierKeys.Shift != 0` instead.

Reply:
Changed made